skill-tree/src/main/java/com/RDS/skilltree/Endorsement/EndorsementServiceImpl.java

```diff
@@ -113,18 +110,17 @@ private Page<EndorsementModelFromJSON> createPagedEndorsements(
 
     @Override
     public EndorsementModel createEndorsement(EndorsementDRO endorsementDRO) {
-        UUID userId = endorsementDRO.getUserId();
+        UUID userId = endorsementDRO.getEndorseeId();
         UUID skillId = endorsementDRO.getSkillId();
-        Optional<UserModel> userOptional = userRepository.findById(userId);
+
```

Comment:
Hi @bhtibrewal - Is there any todo here where we check if the user is present by making call to website backend?

Reply:
Also, can't the userId and skillId be empty. Am i missing if we are validating that case somewhere? I see a Notnull annotation on EndorsementDTO class members but that doesn't take care of "" string.

Reply:
added the follow-up in the description
Yes we verify not null